src/main/java/org/opentripplanner/routing/graphfinder/PlaceFinderTraverseVisitor.java

```diff
@@ -80,6 +81,8 @@ public PlaceFinderTraverseVisitor(
     includeVehicleRentals = shouldInclude(filterByPlaceTypes, PlaceType.VEHICLE_RENT);
     includeCarParking = shouldInclude(filterByPlaceTypes, PlaceType.CAR_PARK);
     includeBikeParking = shouldInclude(filterByPlaceTypes, PlaceType.BIKE_PARK);
+    includeStations = shouldInclude(filterByPlaceTypes, PlaceType.STATION);
```

Comment:
We need special handling for this as we don't want to return stations by default.

This fact needs to also documented in the query docs.

Reply:
Is there some better way to do so than to simply add a check for whether `filterByPlaceTypes` is empty?  Where exactly are the query docs that should be changed?

Reply:
No, that's fine.

The docs to mentioned this are here: https://github.com/opentripplanner/OpenTripPlanner/blob/7b48c4d664a55790e106b6c1028fd69d04d584b6/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls#L2503-L2506

Reply:
Please also add a test for this slightly strange behaviour.

Reply:
Please make sure that stations are only included if you explicitly enable them.